src/main/java/bio/terra/cli/app/LocalProcessCommandRunner.java

```diff
@@ -56,10 +56,14 @@ protected String wrapCommandInSetupCleanup(List<String> command) {
    */
   protected int runToolCommandImpl(String command, Map<String, String> envVars)
       throws PassthroughException {
-
-    // Don't enforce ADC for tests. (`gcloud auth activate-service-account` requires a key file,
-    // which we don't want for security reasons.)
-    if (System.getProperty(CommandRunner.IS_TEST) != null) {
+    if (System.getProperty(CommandRunner.IS_TEST).equals("true")) {
+      // For unit tests, set CLOUDSDK_AUTH_ACCESS_TOKEN. This is how to programmatically
+      // authenticate as test user, without SA key file
+      // (https://cloud.google.com/sdk/docs/release-notes#cloud_sdk_2).
```

Comment:
They finally documented CLOUDSDK_AUTH_ACCESS_TOKEN, can you change url to https://cloud.google.com/sdk/docs/authorizing